symfony/plugins/orangehrmTimePlugin/Api/ProjectAPI.php

```diff
@@ -310,10 +321,17 @@ protected function getCommonBodyValidationRules(): array
 
     /**
      * @inheritDoc
+     * @throws BadRequestException
      */
     public function delete(): EndpointResult
     {
         $ids = $this->getRequestParams()->getArray(RequestParams::PARAM_TYPE_BODY, CommonParams::PARAMETER_IDS);
+        foreach ($ids as $id) {
+            $hasTimesheetItems = $this->getProjectService()->getProjectDao()->hasTimesheetItems($id);
```

Comment:
This need to handle in FE side as well like we handled in Reporting Method

Reply:
Noted